python_modules/dagster/dagster_tests/general_tests/check_tests/test_check.py

```diff
@@ -464,6 +464,35 @@ def test_opt_str_param():
         check.opt_str_param(1, "str_param")
 
 
+def test_path_param():
+    from pathlib import Path
+
+    assert check.path_param("/a/b.csv", "path_param") == "/a/b.csv"
+    assert check.path_param(Path("/a/b.csv"), "path_param") == "/a/b.csv"
+
+    with pytest.raises(ParameterCheckError):
+        check.path_param(None, "path_param")
+
+    with pytest.raises(ParameterCheckError):
+        check.path_param(0, "path_param")
+
+    with pytest.raises(ParameterCheckError):
+        check.path_param(1, "path_param")
+
+
+def test_opt_path_param():
+    from pathlib import Path
+
+    assert check.opt_path_param("/a/b.csv", "path_param") == "/a/b.csv"
```

Comment:
Add a case for `None`

Reply:
Good call, found a bug when the default is PathLike.